man/systemd.link.xml

```diff
@@ -1029,6 +1029,25 @@
           <xi:include href="version-info.xml" xpointer="v250"/>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ReceiveFCS=</varname></term>
+        <listitem>
+          <para>Takes a boolean. If set to true, pass the FCS value up the stack without trimming it, and
```

Comment:
what is FCS? i really have no idea. Please at least say what the acronym's letters mean...

Reply:
Frame Check Sequence. Updated the man page to mention that.